tensorflow/compiler/mlir/hlo/tests/lhlo-input-inline-fusion.mlir

```diff
@@ -0,0 +1,81 @@
+// RUN: mlir-hlo-opt %s -lhlo-legalize-roots-to-parallel-loops -input-inline-fusion -canonicalize -cse -split-input-file | mlir-hlo-opt | FileCheck %s
```

Comment:
In general we're looking for tests that are decoupled: each pass having it's own test: can you split it between the two passes.
It also happens that sometimes running canonicalize may make the output cleaner to match, but if you can avoid it'd be best,

(I'm also not sure why you're piping into `mlir-hlo-opt` a second time here)

Reply:
done.